keras_cv/models/stable_diffusion/stable_diffusion.py

```diff
@@ -228,6 +233,216 @@ def generate_image(
         decoded = ((decoded + 1) / 2) * 255
         return np.clip(decoded, 0, 255).astype("uint8")
 
+    def text_to_image_prompt_to_prompt(
+        self,
+        prompt: str,
+        prompt_edit: str,
+        method: str,
+        self_attn_steps: Union[float, Tuple[float, float]],
```

Comment:
attn->attention, no need to abbreviate this.

Reply:
Dammit, I use attn (almost) everywhere (e.g. `cross_attn_steps`, `attn_edit_weights`, `update_cross_attn_mode`, ...)
Do you really want me to change to to attention? :stuck_out_tongue: